library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -67,7 +67,7 @@ data class ClientOptions(
     val enableV3: Boolean = false,
     val dbDirectory: String? = null,
     val dbEncryptionKey: ByteArray? = null,
-    val historySyncUrl: String? = null,
+    val historySyncUrl: String = "https://message-history.dev.ephemera.network/",
```

Comment:
Should this point to prod when set to prod?
Do we need additional checks like local env?

Reply:
Yep already fixed. 👍